app/src/main/java/at/roteskreuz/stopcorona/model/repositories/NotificationsRepository.kt

```diff
@@ -201,6 +207,21 @@ class NotificationsRepositoryImpl(
         notificationManager.cancel(id)
     }
 
+    override fun displayNotificationForLowRisc() {
+        val title = context.string(R.string.local_notification_low_risk_title)
+        val message = context.string(R.string.local_notification_low_risk_message)
+
+        buildNotification(
+            title = title,
+            message = message,
+            pendingIntent = buildPendingIntentWithActivityStack {
+                addNextIntent(context.getDashboardActivityIntent().addFlags(firstActivityFlags))
+            },
+            channelId = NotificationChannels.CHANNEL_INFECTION_MESSAGE,
+            ongoing = true
```

Comment:
Do we want to have not cancelable notification?

Reply:
no - fixed it.